### PR TITLE
Changes based on end to end testing of Yogi2

### PR DESCRIPTION
**Pull request by mmcadamsms**

### CODE REVIEW

**Comment by mmcadamsms** [comment]
Here are the changes right up to the competition.  Usually what I've done is merge to main prior to competition and then create a competition branch for the weekend so that we can make changes as needed during competition.  Up for suggestions too.

File: yogi2/src/main/java/frc/robot/ManualInputInterfaces.java [hunk 1]
```diff
@@ -166,11 +166,13 @@ private void bindCommandsToDriverXboxButtons()
               this.getGtaInputArcadeDriveX(),
               PowerFactorIncrementDirection.Up),
             new ButtonPress("driverController.kRightBumper", "bumperRight.whenPressed")));
+            /*
```
Comment (josieammer) [inline yogi2/src/main/java/frc/robot/ManualInputInterfaces.java:169]: add a comment about why this is commented?

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java [hunk 2]
```diff
@@ -69,6 +69,10 @@ public void retrieve()
   */
   public void store()
   {
+    //TODO: check the ballStoreSpeed and ballRetrieveSpeeds are the right polarity.
```
Comment (josieammer) [inline yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:72]: Lines 72 and 73 can be removed

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java [hunk 3]
```diff
@@ -69,6 +69,10 @@ public void retrieve()
...
+    // Owen suspects they may be reveresed.  
+
+    // TODO - not sure comment below makes sense?
```
Comment (josieammer) [inline yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:75]: lines 75-76 can be removed

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java [hunk 4]
```diff
@@ -86,6 +90,7 @@ public void setDefaultCommand(Command myCommand)
   */
   public void storeBallManual()
   {
+      // TODO - not sure comment below makes sense?
```
Comment (josieammer) [inline yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:93]: lines 93-94 can be removed

File: yogi2/src/main/java/frc/robot/subsystems/DriveTrain.java [hunk 5]
```diff
@@ -53,8 +53,8 @@ public class DriveTrain extends SubsystemBase implements Sendable
   private MotorControllerGroup left = new MotorControllerGroup(leftFront, leftRear);
   private MotorControllerGroup right = new MotorControllerGroup(rightFront, rightRear);
   private DifferentialDrive forwardDrive = new DifferentialDrive(left, right);
-  private DifferentialDrive reverseDrive = new DifferentialDrive(right, left);
-
+//  private DifferentialDrive reverseDrive = new DifferentialDrive(right, left);
```
Comment (josieammer) [inline yogi2/src/main/java/frc/robot/subsystems/DriveTrain.java:56]: put a comment about why this is commented out (and for lines below)

**Review by josieammer — APPROVED** [review]
Few minor suggestions to add some comments, specifically around the parts of code we commented out.  Going to approve, and you can decide if you want to change these now, or on Monday after the competition.  

Agree with your plan to merge to main before the competition, and create a new branch for changes during the competition.

File: yogi2/src/main/java/frc/robot/ManualInputInterfaces.java (hunk 1 above)
Comment (mmcadamsms) [reply yogi2/src/main/java/frc/robot/ManualInputInterfaces.java:169]: agree

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java (hunk 2 above)
Comment (mmcadamsms) [reply yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:72]: agree

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java (hunk 4 above)
Comment (mmcadamsms) [reply yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:93]: agree

File: yogi2/src/main/java/frc/robot/subsystems/DriveTrain.java (hunk 5 above)
Comment (mmcadamsms) [reply yogi2/src/main/java/frc/robot/subsystems/DriveTrain.java:56]: agree

File: yogi2/src/main/java/frc/robot/subsystems/BallStorage.java (hunk 3 above)
Comment (mmcadamsms) [reply yogi2/src/main/java/frc/robot/subsystems/BallStorage.java:75]: agree

**Pull request merged**